packages/e2e/test/environment.ts

```diff
@@ -1,29 +1,84 @@
 import * as envalid from 'envalid';
 
-export const getEnv = (override: NodeJS.ProcessEnv = {}) =>
+/**
+ * Shared collection of all the environment configuration variables with their constraints
+ */
+const variablesConfiguration = {
+  ASSET_PROVIDER: envalid.str(),
+  ASSET_PROVIDER_PARAMS: envalid.json({ default: {} }),
```

Comment:
You should be able to define the schema for the expected JSON for any of the `envalid.json` declarations. It would be good to cover that improvement in this PR too.